project/Smithy4sPlugin.scala

```diff
@@ -98,7 +98,8 @@ object Smithy4sPlugin extends AutoPlugin {
     incOptions := incOptions.value.withLogRecompileOnMacro(false),
     // https://scalacenter.github.io/scalafix/docs/users/installation.html
     semanticdbEnabled := true,
-    semanticdbVersion := scalafixSemanticdb.revision
+    semanticdbVersion := scalafixSemanticdb.revision,
+    testFrameworks += new TestFramework("weaver.framework.CatsEffect")
```

Comment:
we should submit this to sbt tbh

Reply:
or not, since we'd need all the variants and ain't nobody got time for dat

Reply:
I mean, we could submit just the CatsEffect tbh